pkg/capacityscheduling/capacity_scheduling.go

```diff
@@ -64,7 +64,13 @@ type CapacityScheduling struct {
 
 // PreFilterState computed at PreFilter and used at PostFilter or Reserve.
 type PreFilterState struct {
-	framework.Resource
+	podReq framework.Resource
+	// nominatedPodsReqInEQWithPodReq is the sum of the requested resources of the Nominated Pods
+	// which subject to the same quota(namespace) and podReq.
+	nominatedPodsReqInEQWithPodReq framework.Resource
+	// nominatedPodsReqWithPodReq is the sum of the requested resources of the Nominated Pods
+	// which subject to the all quota(namespace) and podReq.
```

Comment:
Actually, you should also mention only the "guaranteed" nominated pod that don't exceed their min quota are counted here. Because the logic checks `!info.usedOverMin()`.